tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -4190,4 +4190,110 @@ public function test_add_currency_conversion_notice_reads_from_order_meta_withou
+	 * for card and link when Link is enabled, while leaving OC unaffected.
+	 *
+	 * @param string $payment_method_class The payment method class name.
+	 * @param array  $enabled_methods      Enabled UPE payment method IDs.
+	 * @param string $saved_cards          The 'saved_cards' setting value.
+	 * @param bool   $expected             Expected result.
+	 * @return void
+	 *
+	 * @dataProvider provide_test_should_upe_payment_method_show_save_option
+	 */
+	public function test_should_upe_payment_method_show_save_option( $payment_method_class, $enabled_methods, $saved_cards, $expected ) {
+		$settings                = WC_Stripe_Helper::get_stripe_settings();
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'test_should_upe_payment_method_show_save_option'. (undefined)

(StaticAccess)